version.php

```diff
@@ -25,7 +25,7 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version  = 2022030301;  // The current module version (Date: YYYYMMDDXX).
+$plugin->version  = 2022092200;  // The current module version (Date: YYYYMMDDXX).
```

Comment:
I apologize that I didn't notice this before, but the version bump needs to retain the '20220303' portion. This date portion represents the MOODLE_400_STABLE branch.
The MOODLE_401_STABLE branch, if it becomes needed, will have a greater than '20220303' portion.
I will change this. I hope it doesn't negatively impact your installs. If it does, let me know, and I'll release a 4.01 branch with a higher version.

Reply:
I take it back. Since I haven't released MOODLE_400_STABLE yet, I'll just adopt the '20220922' as the base version number for this branch.